tests/sharingTest.js

```diff
@@ -883,7 +883,7 @@ describe('Main: Currently testing file/folder sharing,', function () {
               ...validAuthHeaders,
               ...applicationFormUrlEncoded
             },
-            body: 'shareType=0&shareWith=' + sharee + '&path=' + fileEncoded + '&expireDate=' + expirationDate
+            body: 'shareType=0&shareWith=' + sharee + '&path=' + fileEncoded + '&expireDate=' + getExpirationDate()
```

Comment:
call the date generator function somewhere above and use it everywhere below:
```js
const expirationDate = generateExpirationDate()
```
This way we will use same expirationDate for request and response.
```suggestion
            body: 'shareType=0&shareWith=' + sharee + '&path=' + fileEncoded + '&expireDate=' + expirationDate
```